### DISCUSSION TITLE
WMS Language Requirement error with GeoServer Internationalization capability

### DISCUSSION THREAD

**Discussion by marianhaluska** [INSPIRE-MIF/helpdesk-validator#984] [Q&A]
Hello everyone,

I've been triing to identify a problem I'am facing. Recently we upgraded to newer version of GeoServer (2.21.2) that supports Internationalization (returning a GetCapabilities document in various languages). I did set language versions to support English and Slovak languages but it seems that there is some problem with the funcionality.

I'am sending WMS Capability dokument URL that was tested: https://geo.shmu.sk/geoserver/am-aqmz/ows?request=GetCapabilities&service=WMS&version=1.3.0 

As far as I was able to determine, the problem seems to by in the version of ISO standard that GeoServer uses for its encoding. Is there anyone that has expirienced this problem and could point me to a possible solution.

**Comment by fabiovinci** [comment]
Dear @marianhaluska,

we will take a look at your service and try to give you feedback.

**Comment by lorincztamas** [comment]
Hello!

I have the same problem with geoserver 2.22.4
(build 18-Jun-2023 11:14) when upgrading English and Hungarian language in "internationalization".

I was wondering if there is a solution yet.

Thanks
Tamás

**Comment by fabiovinci** [comment]
Dear all,

I had a look at the issue and it seems that it is related to GeoServer and not to the Validator, so I will move this issue in the Discussions section.

I saw your issue in the dedicated repository (https://osgeo-org.atlassian.net/browse/GEOS-10852), we hope there will soon be a solution.

**Comment by lorincztamas** [comment]
Hi,
Since there is no development capacity on the geoserver side to fix the bug, I revert to an old version 2.19.5 where the language settings are accepted by the validator :(
Anyone have any ideas for other solutions?

Thanks,
Tamás

**Reply by marianhaluska** [reply]
Hi,
I was able to find a ticet on GeoServer that is deeling with similar problem (I think), so I'm sending a link to that thred [HERE](https://osgeo-org.atlassian.net/browse/GEOS-10557). It seems that this problem can be resolved by writing a filter in Java that is then used in web.xml file on GeoServer side but as we have no suficient expiriance with java I was not able to tri this out. Unfortunatly I was naot able to contact Mr. Ralf Trier to ask him for more information. 
Hope it helps somewhat.

Thanks,
Marián
